onyo/lib/commands.py

```diff
@@ -154,7 +154,7 @@ def edit(repo: OnyoRepo,
             repo.git.commit(repo.generate_commit_message(message=message,
                                                          cmd="edit"))
         else:
-            repo.git.restore()
+            repo.git.restore_staged()
```

Comment:
I do remember the idea to change Onyos behavior, so that files get not staged but directly commited, especially because `git diff` can be used before, too.
Here you change a bunch of calls to `git.restore_staged()`. Wouldn't all of these need to be then changed again?

Reply:
Yes. But: One change at a time. This can be resolved when all the widespread staging is consolidated (see #370). Not what this PR is about.